docs/arlas-tagger-configuration.md

```diff
@@ -75,27 +76,28 @@ docker run -ti -d \
 
 | Environment variable | ARLAS Tagger configuration variable | Default | Description |
 | --- | --- | --- | --- |
-| TAGGING_STATUS_TIMEOUT | arlas-tagger.status-timeout | 3600000 | Delay before tagging status is discarded |
-| KAFKA_CONSUMER_POLL_TIMEOUT | arlas-tagger.kafka-consumer-poll-timeout | 100 | Kafka consumer poll timeout |
-| KAFKA_BATCH_SIZE | arlas-tagger.kafka-batch-size | 10 | Kafka consumer batch size |
-| KAFKA_BROKERS | arlas-tagger.kafka-bootstrap-servers | kafka:9092 | Kafka brokers|
-| KAFKA_CONSUMER_GROUP_ID_TAGREF_LOG | arlas-tagger.kafka-consumer-group-id-tagref-log | tagref_log_consumer_group | Kafka consumer group for `tagref_log` topic |
-| KAFKA_CONSUMER_GROUP_ID_EXECUTE_TAGS | arlas-tagger.kafka-consumer-group-id-execute-tags | execute_tags_consumer_group | Kafka consumer group for `execute_tags` topic |
-| KAFKA_TOPIC_TAGREF_LOG | arlas-tagger.kafka-topic-tagref-log | tagref_log | Kafka topic for tag requests queue (tag log, retained) |
-| KAFKA_TOPIC_EXECUTE_TAGS | arlas-tagger.kafka-topic-execute-tags | execute_tags | Kafka topic for actual tag requests (actually executed, not retained) |
+| TAGGING_STATUS_TIMEOUT | kafka_configuration.status_timeout | 3600000 | Delay before tagging status is discarded |
+| KAFKA_CONSUMER_POLL_TIMEOUT | kafka_configuration.kafka_consumer_poll_timeout | 100 | Kafka consumer poll timeout |
+| KAFKA_BATCH_SIZE | kafka_configuration.kafka_batch_size | 10 | Kafka consumer batch size |
+| KAFKA_BROKERS | kafka_configuration.kafka_bootstrap_servers | kafka:9092 | Kafka brokers|
+| KAFKA_CONSUMER_GROUP_ID_TAGREF_LOG | kafka_configuration.kafka_consumer_group_id_tagref_log | tagref_log_consumer_group | Kafka consumer group for `tagref_log` topic |
+| KAFKA_CONSUMER_GROUP_ID_EXECUTE_TAGS | kafka_configuration.kafka_consumer_group_id_execute_tags | execute_tags_consumer_group | Kafka consumer group for `execute_tags` topic |
+| KAFKA_TOPIC_TAGREF_LOG | kafka_configuration.kafka_topic_tagref_log | tagref_log | Kafka topic for tag requests queue (tag log, retained) |
+| KAFKA_TOPIC_EXECUTE_TAGS | kafka_configuration.kafka_topic_execute_tags | execute_tags | Kafka topic for actual tag requests (actually executed, not retained) |
+| KAFKA_EXTRA_PROPS | kafka_configuration.kafka_extra_properties | - | Additional properties for configuring the kafka client. For instance `kafka_configuration.kafka_extra_properties: ssl.endpoint.identification.algorithm=https` |
```

Comment:
`KAFKA_EXTRA_PROPS ` : the separator for the list of pairs must be documented

Reply:
Done